test/unit/strings-test.js

```diff
@@ -81,7 +81,7 @@ describe('strings', function() {
     });
 
     function timeConversionTest(converter) {
-        let sec = converter(5);
```

Comment:
same comment here - the test is testing real applications for this functionality, we should still be able to pass a number in here, maybe we need a union type in the function?

Reply:
makes sense. It's funny because I had added a union type for some other reason, but didn't revert this change.